### PR TITLE
61 frontend placeholder routes in app

### PR DESCRIPTION
**Pull request by BarryFullStack**
Front end routes, component files, and latest Vite and installed react-router-dom

### CODE REVIEW

File: FE/src/components/Account.jsx
Comment (ljreaux) [inline FE/src/components/Account.jsx:1]: Line 3, where you are adding your token and set token needs to be inside of { } or we will have issues later. This is an issue in a few different files.

File: FE/src/components/EditRecipe.jsx
Comment (ljreaux) [inline FE/src/components/EditRecipe.jsx:1]: Not a big deal but Recipe is misspelled "Reciope"

**Review by ljreaux — CHANGES_REQUESTED** [review]
Overall looks good,  everything works. Once the { } are added to props, I approve.

**Review by ljreaux — APPROVED** [review]
I approve.

File: FE/src/App.jsx [hunk 1]
```diff
@@ -1,5 +1,33 @@
...
+import Login from './components/Login'
+import Account from './components/Account'
+import Admin from './components/Admin'
+
 function App() {
-  return <>Hello Lab Rats</>;
+  const [token, setToken] = useState(null)
+
+  return (
+    <>Hello Lab Rats
```
Comment (amead99) [inline FE/src/App.jsx:17]: Might should wrap this in an header or p tag. Not sure if having unwrapped text will negatively affect anything

**Review by jared23m — APPROVED** [review]
looks good i approve

**Comment by amead99** [comment]
approved!

**Pull request merged**